src/assets/styles/colors.scss

```diff
@@ -9,7 +9,7 @@ $gray-light: #CFDFDC;
 $gray-dark: #666666;
 
 $blue-main: #4269f5;
-$blue-light: #1976D2;
+$blue-light: #0a65bf;
```

Comment:
There are multiple components using this `blue-light` color. Did we want to change all of them?